pkg/version/version.go

```diff
@@ -0,0 +1,19 @@
+package version
+
+import "fmt"
+
+// ComboVersion indicates what version of Combo the binary belongs to
+var ComboVersion string
+
+// GitCommit indicates which git commit the binary was built from
+var GitCommit string
```

Comment:
WYDT about also tracking the Kubernetes version too?

Reply:
Took a stab at tracking the Kubernetes version. What are your thoughts on the approach?

Reply:
Replied in https://github.com/operator-framework/combo/pull/35/files#r750568999.